dev/TabView/TestUI/TabViewPage.xaml.cs

```diff
@@ -110,6 +110,14 @@ private void TabViewTabClosing(object sender, Microsoft.UI.Xaml.Controls.TabView
             }
         }
 
+        private void FirstTab_TabClosing(object sender, Microsoft.UI.Xaml.Controls.TabViewTabClosingEventArgs e)
+        {
+            if (CancelItemCloseCheckBox.IsChecked == true)
+            {
+                e.Cancel = true;
```

Comment:
Nit: could just do `e.Cancel = CancelItemCloseCheckBox.IsChecked == true;`

Reply:
that is SO true